easybuild/easyconfigs/i/iomkl/iomkl-2016.03b.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = "Toolchain"
+
+name = 'iomkl'
+version = '2016.03b'
```

Comment:
@jhein32 I would version this as `2016.07`, to be in line with the release date of the most recent component, which is OpenMPI 1.10.3 in this case (July'16)?